### ISSUE TITLE
Errors loading Refill map on main metro extracts page

### ISSUE THREAD

**Issue by nvkelso** [mapzen/metro-extracts#313]
## tl;dr

- Looks like we're using Tangram v0.10 instead of Tangram latest (or at least v0.13).
    - This is from a very old version of Mapzen.js: https://github.com/mapzen/metro-extracts/blob/cb50cf40ee8d92f3d1467c8679775f60340f3856/App/templates/layout.html#L34
    - Mapzen.js v0.8 series has the required Tangram v0.13:  https://github.com/mapzen/mapzen.js/releases/tag/release-v0.8.1
- Importing an un-versioned Refill which tempts fate against stable Tangram.
    - https://github.com/mapzen/metro-extracts/blob/master/App/static/scene.yaml#L2
- The newer Refill styles require more recent versions of Tangram (like v0.13)
- Missing file color theme during import via relative path is a problem (should use ZIP and newer syntax).
- Instead of building our own scene file we could just use Mapzen.js's named basemap styles.
    - Looks like a custom scene is used to disable the waves in the water (to a flat polygon style).
- Unrelated Mapzen Styleguide bug.

**New syntax** from https://mapzen.com/blog/refill-levels-of-detail-and-labels/:

```
import:
    # basemap style
    - https://mapzen.com/carto/refill-style/8/refill-style.zip
    # modify the basemap via themes
    # already defaults to black color
    # add more labels
    - https://mapzen.com/carto/refill-style/8/themes/label-10.zip
```

**OR**

We ask for RefillMoreLabels using Mapzen.js.

===

Console log contains errors like:

```
Uncaught TypeError: Cannot set property 'innerHTML' of null
at e (mapzen-styleguide.min.js:5)
    at XMLHttpRequest.n.onload 
(Tangram v0.10.6 [error]: ERROR: 0:583: 'rgb' :  field selection requires structure or vector on left hand side
Tangram v0.10.6 [error]: StyleManager.compile(): error compiling style colorized-icons: Error: style.compile(): style colorized-icons error:
Tangram v0.10.6 [error]: StyleManager.compile(): error compiling style building-grid: Error: style.compile(): style building-grid error:
global.lightest_color:1 GET https://mapzen.com/carto/refill-style-more-labels/global.lightest_color 404 (Not Found)
global.lighter_color:1 GET https://mapzen.com/carto/refill-style-more-labels/global.lighter_color https://mapzen.com/carto/refill-style-more-labels/global.black_color 404 (Not Found)
Tangram v0.10.6 [warn]: Texture 'https://mapzen.com/carto/refill-style-more-labels/global.lightest_color': failed to load url: 'https://mapzen.com/carto/refill-style-more-labels/global.lightest_color' 
...
```
